### ISSUE TITLE
rabbitmqctl list_user_permissions call doesn't unescape permission regexes

### ISSUE THREAD

**Issue by tedgin** [rabbitmq/rabbitmq-server#596]
I'm using version 3.3.1 of the product.  

I have  a user named `svc` with a configure privilege regular expression of `^amq\.gen.*`.  When I use rabbitmqctl to list the user's privileges, it shows up as `^amq\\.gen.*`. 

```
? rabbitmqctl list_user_permissions svc
Listing permissions for user "svc" ...
/	^amq\\.gen.*	^$	^$
...done.
```

This is a very minor issue, but it does cause ansible's rabbitmq_user module to unnecessarily set the user's permissions.

**Comment by michaelklishin** [comment]
Sorry but `3.3.x` is no longer being developed. Please try if that's still the case with `3.6.0`.

**Comment by michaelklishin** [comment]
Still present in 3.6.x.

**Comment by dcorbacho** [comment]
Solved in #655, list_permissions was also affected.

It avoids escaping again the full output of the selected commands, but avoids deep changes in the code at this point.
